torch_xla/csrc/ops/dynamic_ir.cpp

```diff
@@ -41,6 +41,9 @@ int64_t SizeNode::getDynamicValue() const {
   }
   torch::lazy::NodePtr cloned =
```

Comment:
What's the reason we need to clone a sizeNode, as opposed to using the current `this` SizeNode?

Reply:
wrapping `self` inside a new object(dummy_size_tensors) and pass it to another function is kind of weird. I also need to deal with const_cast if I do it that way. I think it is just easier to make a clone and no worry about poetical consequence.